samples/snippets/cpp/VS_Snippets_Misc/NVC_MFC_VisualStudioDemo/cpp/VisualStudioDemo.rc

```diff
@@ -185,7 +185,7 @@ FONT 8, "MS Sans Serif", 0, 0, 0x1
 BEGIN
     ICON            IDR_MAINFRAME,IDC_STATIC,11,17,20,20
     LTEXT           "VisualStudioDemo Application",IDC_STATIC,40,10,119,8
-    LTEXT           "� 2007 - 2008 Microsoft Corporation.  All rights reserved.",
+    LTEXT           "� 2007 - 2008 Microsoft Corporation.  All rights reserved.",
```

Comment:
Something went wrong here.

Reply:
Good catch! Probably an auto conversion to UTF-8. Do RC files need to be marked as windows encoding for git?

Reply:
Copied back the old character while the file was marked as UTF8 encoded, so there is a diff, but no longer the question mark character